apps/books/templates/categoryvisebooks.html

```diff
@@ -0,0 +1,28 @@
+<h1>{{ object.title }}:</h1>
+<ul style="list-style-type: none;">
+    {% for book in object.book_set.all %}
```

Comment:
we can add related_name in models to use `books` instead of `book_set` ? WDYT? Please use `related_name='books'` in Book model